components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/AuthorizationHandlerManager.java

```diff
@@ -129,6 +129,22 @@ private OAuth2AuthorizeRespDTO validateAuthzRequest(OAuth2AuthorizeReqDTO authzR
             return authorizeRespDTO;
         }
 
+        List<String> allowedScopes = OAuthServerConfiguration.getInstance().getAllowedScopes();
+        List<String> requestedAllowedScopes = new ArrayList<>();
+        String[] requestedScopes = authzReqMsgCtx.getAuthorizationReqDTO().getScopes();
+        List<String> requestedScopesList = new ArrayList<>();
```

Comment:
rename requestedScopesList to scopesToValidate or scopesToBeValidated

Reply:
renamed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1468/commits/760fadc552937ae7aee54dd69d30696c39f0f05f